dependencies.py

```diff
@@ -76,6 +76,7 @@ def extra_requirements() -> dict[str, list[str]]:
         "tdengine": ["taos-ws-py==0.3.2"],
         "snowflake": ["snowflake-connector-python~=3.7"],
         "openai": ["openai~=1.88"],
+        "huggingface": ["huggingface-hub~=0.33.0"],
```

Comment:
`huggingface-hub~=0.33.0` means `huggingface-hub==0.34.0` won't be compatible, see:
https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean

```suggestion
        "huggingface": ["huggingface-hub>=0.33.4, <1"],
```

Reply:
fixed to `huggingface-hub~=0.33`